src/main/java/teammates/ui/automated/FeedbackSessionRemindParticularUsersEmailWorkerAction.java

```diff
@@ -34,11 +34,14 @@ public void execute() {
         String feedbackSessionName = getNonNullRequestParamValue(ParamsNames.SUBMISSION_FEEDBACK);
         String courseId = getNonNullRequestParamValue(ParamsNames.SUBMISSION_COURSE);
         String[] usersToRemind = getNonNullRequestParamValues(ParamsNames.SUBMISSION_REMIND_USERLIST);
+        String[] googleIdOfInstructorToNotify = getNonNullRequestParamValues(ParamsNames.USER_ID);
```

Comment:
Does this need to be a `String[]` (i.e. an array)?

Reply:
Just realized there is a method to get only the [0] index of String[] params. Updated